test/CMakeLists.txt

```diff
@@ -73,4 +73,4 @@ target_include_directories(test-fisco-bcos PRIVATE ..)
 target_include_directories(test-fisco-bcos PRIVATE ../utils)
 target_include_directories(test-fisco-bcos PRIVATE ${ROCKSDB_INCLUDE_DIR})
 target_link_libraries(test-fisco-bcos Boost::UnitTestFramework)
-target_link_libraries(test-fisco-bcos initializer)
+target_link_libraries(test-fisco-bcos initializer channelserver)
```

Comment:
why this? I think this should not add

Reply:
The link will be failed of test-fisco-bcos when compiling on linux if not add the `channelserver` here.

Reply:
ok